docs/source/index.rst

```diff
@@ -1,6 +1,8 @@
 
-llvmlite -- a light-weight Python binding to LLVM
-=================================================
+llvmlite
+========
+
+**A lightweight LLVM python binding for writing JIT compilers**
```

Comment:
Nvm, just realize that description is already used in the README.  So, it's fine.

Perhaps, this https://github.com/numba/numba/pull/2123#issuecomment-246792878 applies here as well.